classy/gui.py

```diff
@@ -5,10 +5,14 @@
 import classy
 from classy.log import logger
 
-import pyqtgraph as pg
-
-# from pyqtgraph.Qt import QtCore, QtWidgets
-from pyqtgraph.Qt import QtWidgets, QtGui
+try:
+    import pyqtgraph as pg
+    from pyqtgraph.Qt import QtWidgets
+except ModuleNotFoundError as e:
```

Comment:
If I have pyqtgraph installed but not a Qt library, this won't trigger because pyqtgraph unfortunately raises an Exception in that case, not ModuleNotFoundError:
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/ari/dev/karman/classy/classy/gui.py", line 9, in <module>
    import pyqtgraph as pg
  File "/home/ari/.cache/pypoetry/virtualenvs/space-classy-yXkZV_lt-py3.11/lib/python3.11/site-packages/pyqtgraph/__init__.py", line 18, in <module>
    from .colors import palette
  File "/home/ari/.cache/pypoetry/virtualenvs/space-classy-yXkZV_lt-py3.11/lib/python3.11/site-packages/pyqtgraph/colors/palette.py", line 1, in <module>
    from ..Qt import QtGui
  File "/home/ari/.cache/pypoetry/virtualenvs/space-classy-yXkZV_lt-py3.11/lib/python3.11/site-packages/pyqtgraph/Qt/__init__.py", line 54, in <module>
    raise Exception("PyQtGraph requires one of PyQt5, PyQt6, PySide2 or PySide6; none of these packages could be imported.")
Exception: PyQtGraph requires one of PyQt5, PyQt6, PySide2 or PySide6; none of these packages could be imported.
```

The current message raised here is actually only raised if you don't have pyqtgraph, but it only mentions pyqt5, so it's sort of hiding the true reason for the exception.

Reply:
Good catch, fixed!